ApiReview/API.Polly.RateLimiting/NoDocs/Polly.RateLimiting/RateLimiterStrategyOptions.cs

```diff
@@ -0,0 +1,18 @@
+// Assembly 'Polly.RateLimiting'
+
+using System;
+using System.ComponentModel.DataAnnotations;
+using System.Runtime.CompilerServices;
+using System.Threading.RateLimiting;
+using System.Threading.Tasks;
+
+namespace Polly.RateLimiting;
+
+public class RateLimiterStrategyOptions : ResilienceStrategyOptions
+{
+    [Required]
+    public ConcurrencyLimiterOptions DefaultRateLimiterOptions { get; set; }
+    public Func<OnRateLimiterRejectedArguments, ValueTask>? OnRejected { get; set; }
+    public ResilienceRateLimiter? RateLimiter { get; set; }
```

Comment:
```suggestion
    public Func<ValueTask<RateLimitLease>, ResilienceContext> RateLimiter { get; set; }
```

Reply:
I think you meant:

` public Func<ResilienceContext, ValueTask<RateLimitLease>> RateLimiter { get; set; }`

Looks good, but we would have to solve the problem with resource management i.e. disposing unused rate limiters after pipeline is reloaded.

Reply:
Give this a shot.